pvlib/test/test_ivtools.py

```diff
@@ -102,6 +109,31 @@ def test_fit_sdm_cec_sam(get_cec_params_cansol_cs5p_220p):
                 cells_in_series=1, temp_ref=25)
 
 
+@requires_scipy
+def test_fit_sdm_desoto(get_test_specs_params):
+    result = ivtools.fit_sdm_desoto(cells_in_series=60,
```

Comment:
move `cell_in_series` to the `get_test_specs_params` fixture.

Reply:
cells_in_series is not in the get_test_specs_params fixture because it changes at line 129 of the test in order to cause the RuntimeError. I feel like it would be more complicated to add it the fixture and to change it later to have the RuntimeError. What do you think ?

Reply:
I'd prefer to see it added to the fixture and then modified in the function that causes the `RuntimeError`. That way the fixture returns a complete, consistent specification.